app/script/view_model/content/PreferencesAVViewModel.coffee

```diff
@@ -74,8 +74,10 @@ class z.ViewModel.content.PreferencesAVViewModel
       return @audio_stream()
     .catch (error) =>
       error = error[0] if _.isArray error
-      @permission_denied true
       @logger.log @logger.levels.ERROR, "Requesting MediaStream failed: #{error.message}", error
+      if error.name in z.calling.rtc.MediaStreamErrorTypes.DEVICE or error.name in z.calling.rtc.MediaStreamErrorTypes.PERMISSION
+        @permission_denied true
```

Comment:
`error.name in [z.calling.rtc.MediaStreamErrorTypes.DEVICE, z.calling.rtc.MediaStreamErrorTypes.PERMISSION]`